Makefile

```
@@ -0,0 +1,25 @@
+all: flake isort pipenv_check django_migrations_linter dotenv_linter vulture
+
+flake:
+	@echo "Flake8"
+	-@flake8
```

do not use `-` when running linte. `-` it makes `make` to ignore exit code of command, in this case `flake8`:

```suggestion
	@flake8
```